packages/admin/codegen.yml

```diff
@@ -0,0 +1,13 @@
+overwrite: true
+schema: '../backend/src/graphql/schema.graphql'
```

Comment:
It's better to do this in the existing codegen file in the backend, like we did with the map. This is because if a developer changes the schema on the backend they don't have to remember to update the generated files in multiple packages.